### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by AkihiroSuda**
* remove unmaintained manifest.md
* add usage to README.md
* regenerate AUTHORS

Signed-off-by: Akihiro Suda <suda.akihiro@lab.ntt.co.jp>

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
@stevvooe PTAL

File: manifest.md [hunk 1]
```diff
@@ -1,75 +0,0 @@
-
-A _bundle manifest_ contains metadata to allow recreation of filesystem images
```
Comment (stevvooe) [inline manifest.md]: Rather than deleting, could you update?

**Comment by stevvooe** [comment]
LGTM

**Comment by AkihiroSuda** [comment]
@dmcgowan PTAL

**Pull request merged**